jobs/uaa/spec

```diff
@@ -744,6 +744,16 @@ properties:
     description: "how long the results of a preflight request is cached"
     default: ~
 
+  # Content Security Policy (csp) settings
+  uaa.csp.script-src:
+    description: "Overrides the default script-src CSP header value of 'self'. Set this to allow scripts to be loaded from sources besides UAA. Because this overrides (not appends to) the value, be sure to include an entry for 'self' so scripts from UAA are allowed."
+    default:
+      - "'self'"
```

Comment:
I am worried about the syntax here requiring a double quote followed by a single quote `"'self'"`. Are the single quotes really required? I'm afraid this will cause problems.

Reply:
yes they are needed because of syntax described from https://content-security-policy.com/script-src